.circleci/config.yml

```diff
@@ -134,18 +134,24 @@ jobs:
             docker tag cimg/base:20.04 cimg/base:current
```

Comment:
Should this be `docker tag cimg/base:22.04 cimg/base:current` if Jammy is moving to current?